### ISSUE TITLE
The training results are inconsistent with the paper, are there other parameters that need to be modified？

### ISSUE THREAD

**Issue by Shirugo** [liuquande/SRC-MT#5]
Hi！
Thanks for sharing your great work! I have some questions to ask you.Thank you very much!I used the chestx-ray14 dataset for training with a batch-size of 48 and an epoch of 20. At the same time,I used the pre-trained model of densenet169 mentioned in your paper, but the final result sensitivity is 0.072 and AUROC is 0.51, which is quite different from the result of your paper.Have you made any special changes to the parameters?
Best regards！

**Comment by ymsunsjtu** [comment]
Have you ever solved the problem? I have the same problem with the AUC is 0.58 and sensitivity with 0.18.

**Comment by Shirugo** [comment]
Yes, I have always been confused. This problem has not been solved. I am also waiting for the reply of the author. This is my email. If you can find the problem and solve it, please let me know! thank you!




At 2021-11-11 13:35:55, "ymsunsjtu" ***@***.***> wrote:

Have you ever solved the problem? I have the same problem with the AUC is 0.58 and sensitivity with 0.18.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Cgxdgfcd** [comment]
Have you ever solved the problem? I'm also confused about this.
